lib/puppet/provider/ibm_pkg/imcl.rb

```diff
@@ -77,7 +77,7 @@ def installed_file
     if resource[:user] == 'root'
       xml_path = '/var/ibm/InstallationManager/installed.xml'
     else
-      xml_path = "/home/#{resource[:user]}/var/ibm/InstallationManager/installed.xml"
+      xml_path = "#{resource[:user_home]}/var/ibm/InstallationManager/installed.xml"
```

Comment:
this is a breaking change. if only `user` is passed and is not 'root', `the xml_path` will be '/var/ibm/InstallationManager/installed.xml'. seems like this could be fixed by checking if user_home is set.